pkg/operator/operator.go

```diff
@@ -89,9 +90,10 @@ func init() {
 type Operator struct {
 	manager.Manager
 
-	KubernetesInterface kubernetes.Interface
-	EventRecorder       events.Recorder
-	Clock               clock.Clock
+	KubernetesInterface   kubernetes.Interface
+	ApiextensionInterface clientset.Interface
```

Comment:
This isn't needed anymore, right? Further, I don't think that we even need the KubernetesInterface in the operator anymore, but we can take that change separately